pennylane_lightning/core/src/simulators/lightning_kokkos/measurements/MeasurementsKokkos.hpp

```diff
@@ -194,7 +193,9 @@ class Measurements final
      * @param ob HermitianObs.
      * @return Expectation value with respect to the given observable.
      */
-    PrecisionT expval(const HermitianObs<StateVectorT> &ob) {
+    PrecisionT
+    expval(const Pennylane::LightningKokkos::Observables::HermitianObs<
```

Comment:
I'm curious, does this conflict with a symbol in one of the headers?

Reply:
:thinking:

Reply:
Yes, it was conflicting with symbols in lightning-qubit!